cas-server-support-spnego/src/main/java/org/jasig/cas/support/spnego/authentication/handler/support/JcifsConfig.java

```diff
@@ -107,24 +112,30 @@ public void afterPropertiesSet() throws Exception {
         logger.debug("configured login configuration path : {}", propValue);
     }
 
+
     /**
      * Sets the jcifs service password.
      *
      * @param jcifsServicePassword the new jcifs service password
      */
     public void setJcifsServicePassword(final String jcifsServicePassword) {
-        logger.debug("jcifsServicePassword is set to *****");
-        Config.setProperty(JCIFS_PROP_SERVICE_PASSWORD, jcifsServicePassword);
+        if (StringUtils.isNotBlank(jcifsServicePassword)) {
```

Comment:
Setters are invoked via autowiring to adjust properties, based on cas.properties. We want to support defaults, where the default sometimes might be blank. If this check isnt there, the config property will be set to blank, which might cause issues with "the real default". So we only set explicit values if one is allowed, otherwise we leave it as be.